### ISSUE TITLE
Issue when fetching range of coordinates

### ISSUE THREAD

**Issue by danielegrattarola** [jagoosw/gfspy#2]
Hi, 

I have encountered a bug in the library which occurs when fetching a range of coordinates instead of just a point.

By running:

```python
>>> f = getgfs.Forecast("1p00")
>>> res = f.get(["gustsfc"], "20220509 5:30", "[-90:90]", "'[0:360]'")
```

We get
```python
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Input In [49], in <module>
----> 1 res = f.get(["gustsfc"], "20220509 5:30", "[-90:90]", "'[0:360]'")

File ~/dev/env/lib/python3.9/site-packages/getgfs/getgfs.py:103, in Forecast.get(self, variables, date_time, lat, lon)
    100 forecast_date, forecast_time, query_time = self.datetime_to_forecast(date_time)
    102 # Get latitude
--> 103 lat = self.value_input_to_index("lat", lat)
    105 # Get longitude
    106 lon = self.value_input_to_index("lon", lon)

File ~/dev/env/lib/python3.9/site-packages/getgfs/getgfs.py:259, in Forecast.value_input_to_index(self, coord, inpt)
    257 if isinstance(inpt, str):
    258     if inpt[0] == "[" and inpt[-1] == "]" and ":" in inpt:
--> 259         val_1 = float(re.findall(r"\[(.*?):", inpt))
    260         val_2 = float(re.findall(r"\:(.*?)]", inpt))
    261         val_min = self.value_to_index(coord, min(val_1, val_2))

TypeError: float() argument must be a string or a number, not 'list'
```

The reason for the bug is that ```re.findall``` returns a list, not a string, and therefore the casting to float fails. 
By changing lines 259 and 260 of `getgfs/getgfs.py` to: 

```python
val_1 = float(re.findall(r"\[(.*?):", inpt)[0])
val_2 = float(re.findall(r"\:(.*?)]", inpt)[0])
```

the issue is solved.

**Comment by jagoosw** [comment]
Hi,

Thanks for bringing this up, more than happy to merge that in if you make the change or I can do it in a couple of days?

Thanks,
Jago

**Comment by jagoosw** [comment]
Fixed by #5
